lte/cloud/go/services/nprobe/nprobe/main.go

```diff
@@ -62,7 +63,22 @@ func main() {
 	protos.RegisterSwaggerSpecServer(srv.GrpcServer, swagger.NewSpecServicerFromFile(nprobe.ServiceName))
 
 	serviceConfig := nprobe.GetServiceConfig()
-	nProbeManager, err := manager.NewNProbeManager(serviceConfig)
+	tlsConfig, err := exporter.NewTlsConfig(
+		serviceConfig.ExporterCrtFile,
+		serviceConfig.ExporterKeyFile,
+		serviceConfig.SkipVerifyServer,
+	)
+	if err != nil {
+		glog.Errorf("Failed to create new TlsConfig: %v", err)
+	}
+
+	// Init records exporter
+	recordExporter, err := exporter.NewRecordExporter(serviceConfig.DeliveryFunctionAddr, tlsConfig)
```

Comment:
why is the error message prompting a log but not having any effect here? Does it make any sense to run this if `tlsConfig` is nil?